include/nil/blueprint/blueprint/plonk/assignment.hpp

```diff
@@ -131,6 +131,10 @@ namespace nil {
                 }
             }
 
+            virtual void fill_selector(std::uint32_t index, const column_type& column) {
```

Comment:
I would probably remove both of these from the file: they should be no longer be useful after zk changes, just a wrapper layer.

Reply:
done